pkgs/shelf_router/lib/src/router.dart

```diff
@@ -190,6 +190,21 @@ class Router {
     return _notFoundHandler(request);
   }
 
+  /// Get URL parameters captured by the [Router].
+  /// Returns `null` if no parameters are captured.
```

Comment:
```suggestion
  /// Get URL parameters captured by the [Router].
  ///
  /// Returns `null` if no parameters are captured.
```

https://dart.dev/effective-dart/documentation#do-separate-the-first-sentence-of-a-doc-comment-into-its-own-paragraph